src/pages/128x64x1/advanced/mixer_curves.c

```diff
@@ -95,9 +95,10 @@ void PAGE_EditCurvesInit(int page)
 static unsigned action_cb(u32 button, unsigned flags, void *data)
 {
     (void)data;
-    if ((flags & BUTTON_PRESS) || (flags & BUTTON_LONGPRESS)) {
+    if ((flags & BUTTON_PRESS) || (flags & BUTTON_LONGPRESS) || (flags & BUTTON_RELEASE)) {
         if (CHAN_ButtonIsPressed(button, BUT_EXIT)) {
-            PAGE_Pop();
+            if (flags & BUTTON_RELEASE) 
```

Comment:
please remove the extra space at the end of the line so the linter stops complaining

Reply:
OK, strange that I cannot see the linter errors 🤔

Reply:
it's down in the bottom at the GitHub checks or just run make lint